### PR TITLE
docs(aad): add doc for aad.AadGetAppError

### PR DESCRIPTION
**Pull request by KennethBWSong**

### CODE REVIEW

File: docs/fx-core/aad-help.md [hunk 1]
```diff
@@ -135,4 +135,24 @@ Root cause of this error is that necessary info is not included in env.default.j
...
+Root cause of this error is that toolkit can not find Azure AD app with the same object id saved in your "env.default.json" file. Please follow the instruction following to address the error.
+
+1. Open `.fx\env.default.json` file
+2. Find `fx-resource-aad-app-for-teams`. Note value of key *clientId* or *local_clientId*
+3. Go to Azure Portal, login with the same account as the M365 account in toolkit, select "Azure Active Directory"
+4. Select "App Registrations" and search for you Azure AD app by client id noted above.
+
+If you can find your Azure AD app, please check your network status and try again.
+
+If you can not find your Azure AD app, please check whether you logged in with the correct account.
```
Comment (SLdragon) [inline docs/fx-core/aad-help.md:158]: Do we need to add another tip that: "You can also remove objectId from env.default.json file"

Comment (KennethBWSong) [reply docs/fx-core/aad-help.md:158]: Fixed. Thanks!

File: docs/fx-core/aad-help.md [hunk 2]
```diff
@@ -135,4 +135,24 @@ Root cause of this error is that necessary info is not included in env.default.j
...
+### Mitigation
+
+Root cause of this error is that toolkit can not find Azure AD app with the same object id saved in your "env.default.json" file. Please follow the instruction following to address the error.
+
+1. Open `.fx\env.default.json` file
+2. Find `fx-resource-aad-app-for-teams`. Note value of key *clientId* or *local_clientId*
+3. Go to Azure Portal, login with the same account as the M365 account in toolkit, select "Azure Active Directory"
+4. Select "App Registrations" and search for you Azure AD app by client id noted above.
+
+If you can find your Azure AD app, please check your network status and try again. If this problem still exists, please remove objectId from env.default.json file and then try again.
```
Comment (adashen) [inline docs/fx-core/aad-help.md:156]: So even if the app is found, the object id could be wrong?

Comment (KennethBWSong) [reply docs/fx-core/aad-help.md]: This scenario should not happen. Sorry I put the message in wrong place. It is fixed now.

**Review by adashen — APPROVED** [review]

**Pull request merged**